src/Components/CookieConsent.js

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+import CookieConsent from "react-cookie-consent";
+
+/**
+ * @return {null}
+ */
+function CookieConsentBox() {
+  const root = Scrivito.Obj.root();
+  const cookieLink = root.get("cookiesPrivacyPolicyLink");
+  const cookiesLinkTitle = cookieLink && cookieLink.title();
+
+  if (!cookieLink) {
+    return null;
+  }
+
+  return (
+    <CookieConsent
+      buttonText="accept"
+      disableStyles="true"
+      containerClasses="cookie-box flex-row d-lg-flex d-sm-flex"
+      contentClasses="cookie-content flex-row d-flex"
+      buttonClasses="btn btn-primary cookie-button ml-auto"
+    >
+      <div className="cookie-img-box">
+        <img
+          className="cookie-img"
+          src="../../src/assets/images/iconmonstr-candy-6.svg"
```

Comment:
Please do not hard-code paths to files. Instead please use imports as provided by the webpack [`file-loader`](https://github.com/webpack-contrib/file-loader) plugin. It works like this:

You simply import the file like this:

```js
import cookieConsentIcon from "../assets/images/iconmonstr-candy-6.svg";
```

And than you can use the filename as this:

```jsx
<img className="cookie-img" src={cookieConsentIcon} />
```

Please note, that at least on my machine I had to restart the webpack server after the change.

Reply:
Please also change the file name of `iconmonstr-candy-6.svg`. All other svg files are named snake case. I suggest a name like `cookie_consent_icon.svg`.

Also one more thing: What is the source of the icon? Is it public domain? Or do we need to give credit for it?